src/main/java/dev/codesupport/web/api/config/ApplicationConfiguration.java

```diff
@@ -0,0 +1,18 @@
+package dev.codesupport.web.api.config;
+
+import dev.codesupport.web.common.service.service.CrudOperations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+class ApplicationConfiguration {
+
+    @Autowired
+    ApplicationConfiguration(
+            ApplicationContext context
+    ) {
+        CrudOperations.setContext(context);
+    }
```

Comment:
Why is a constructor being used for something that could very easily be a static procedure on utility class?